src/commands/core/follow.ts

```diff
@@ -0,0 +1,24 @@
+import Command from '../../lib/structures/Command';
+import { TypicalGuildMessage } from '../../lib/types/typicalbot';
+import { Modes, PermissionsLevels } from '../../lib/utils/constants';
+
+export default class extends Command {
+    mode = Modes.STRICT;
+    permission = PermissionsLevels.SERVER_ADMINISTRATOR;
+
+    async execute(message: TypicalGuildMessage, parameters: string) {
+        const [type, id] = parameters.split(' ');
+        const channel = message.mentions.channels.first() ?? message.guild.channels.cache.get(id) ?? message.channel;
+        if (!message.guild.me?.permissions.has('MANAGE_WEBHOOKS'))
```

Comment:
I had `true` after the `MANAGE_WEBHOOKS` because I was having issues getting it to recognize administrator on other commands. Since all the other commands have the `true` after it, lets add it back to keep it consistent.

Reply:
If true is necessary there is something else wrong somewhere. By default, this is `true`
![image](https://user-images.githubusercontent.com/23035000/86862766-dab34d80-c097-11ea-884b-28c47ec6c80a.png)